### PR TITLE
install phpcs via composer

### PR DESCRIPTION
**Pull request by cfoellmann**
phpcs needs to be installed via composer to reduce the provision process significantly. My last `vagrant provision` took __7039__ seconds! mostly due to git clones.

### CODE REVIEW

**Comment by cfoellmann** [comment]
I fixed the oversight of setting the correct path to the WP coding standards.
This should save 5-10 minutes for the first provision.

**Comment by cfoellmann** [comment]
@jeremyfelt please add `in progress` label

**Comment by jeremyfelt** [comment]
Thanks @cfoellmann!

* Is the switch in directories from `/srv/www/phpcs/CodeSniffer/Standards/WordPress` to `/srv/www/phpcs-wordpress` going to mess with any current installations?
* Why `sudo` now on the phpcs lines?

File: provision/provision.sh [hunk 1]
```diff
@@ -476,9 +463,9 @@ if [[ $ping_result == "Connected" ]]; then
 		fi
 	fi
 	# Install the standards in PHPCS
-	/srv/www/phpcs/scripts/phpcs --config-set installed_paths ./CodeSniffer/Standards/WordPress/
-	/srv/www/phpcs/scripts/phpcs --config-set default_standard WordPress-Core
-	/srv/www/phpcs/scripts/phpcs -i
+	sudo /srv/www/phpcs/scripts/phpcs --config-set installed_paths /srv/www/phpcs-wordpress/
```
Comment (westonruter) [inline provision/provision.sh]: This absolute path will break PHPCS when invoked from outside of Vagrant. This is why a relative path is used currently.

**Comment by westonruter** [comment]
The path to the WPCS shouldn't be changed.

**Comment by jeremyfelt** [comment]
@cfoellmann do you want to address the path naming and update the PR accordingly? It seems like this would help as long as we're consistent with the location.

**Comment by jeremyfelt** [comment]
I think we need to regroup on this in a new pull request. I've opened issue #922 to describe the intent here. Thanks for the work on this @cfoellmann. :)

**Pull request closed without merging**